src/sdam/server.ts

```diff
@@ -270,7 +270,6 @@ export class Server extends TypedEventEmitter<ServerEvents> {
     }
 
     if (this.s.state === STATE_CLOSING || this.s.state === STATE_CLOSED) {
-      // TODO(NODE-3405): Change this out for MongoServerClosedError
       callback(new MongoDriverError('Server is closed'));
```

Comment:
SeverClosedErr?

Reply:
This TODO and this one [here](https://github.com/mongodb/node-mongodb-native/pull/2912#discussion_r676945514) have both been changed in [NODE-3405](https://github.com/mongodb/node-mongodb-native/pull/2913) so I removed the TODOs here.